src/main/fc/runtime_config.c

```diff
@@ -23,19 +23,19 @@
 #include "fc/runtime_config.h"
 #include "io/beeper.h"
 
-uint8_t armingFlags = 0;
-uint8_t stateFlags = 0;
-uint16_t flightModeFlags = 0;
+armingFlag_e armingFlags = 0;
+stateFlags_e stateFlags = 0;
+flightModeFlags_e flightModeFlags = 0;
```

Comment:
Me either. Perhaps uint8_t and then use defined constants rather than enum for the flags themselves?